tests/wrapper_tests.cpp

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+// # Author: Jonathan Wong @ LLNL.
+
+#include <gtest/gtest.h>
+
+#include "mfem.hpp"
+#include <memory>
+#include "coefficients/stdfunction_coefficient.hpp"
+#include "integrators/wrapper_integrator.hpp"
+
+using namespace mfem;
```

Comment:
Please don't do this not everyone knows where things are coming from... namely me.

Reply:
Should I change it in my previous test `StdFunctionCoefficient` also?

Reply:
I guess we've had this conversation partially in the past and perhaps it should be discussed further in person. I find it a bit tedious to refer to everything explicitly. I understand why it shouldn't be put into a header, but at some point it is a style preference and trade off.

Would you prefer I do this instead? (the Axom-way)
```
using Vector = mfem::Vector
```

Reply:
intellisense can see through `using` statements to clarify types (when hovering cursor over `u`):

![intellisense](https://user-images.githubusercontent.com/16393433/78840363-617c5f80-79af-11ea-9da1-3c856442a049.png)

I agree `using namespace` shouldn't be in headers, but it's not a big deal otherwise.

Reply:
on an unrelated note: could you also change the variable with name `restrict` in the StdFunctionCoefficient tests to `restricted` or similar? It's a reserved word in C, and seems to confuse my syntax-highlighter!

Reply:
I only like using's when they shorten namespaces (note remove completely) or are limited to a function in very limited quantity.  We can have a conversation about this at a later date in person.

Reply:
To clarify my comment, just leave it for now especially since its a test and not in a header.

Reply:
I am also generally against `using` statements. However, I think tests are fine since they are self contained pieces of code. 

This brings up the fact that we should probably draft and discuss a 'best practices' guide at some point that goes beyond simple formatting. More work for the pile... 😭